apps/site/src/components/canvas/View.tsx

```diff
@@ -1,28 +1,32 @@
 "use client";
 
+import { PropsWithChildren } from "react";
 import { forwardRef, useImperativeHandle, useRef } from "react";
-import { OrbitControls, View as ViewImpl } from "@react-three/drei";
+import { View as ViewImpl } from "@react-three/drei";
 import { Three } from "@/helpers/components/Three";
 
-const View = forwardRef(({ children, orbit, ...props }: any, ref) => {
-	const localRef = useRef<any>();
-	useImperativeHandle(ref, () => localRef.current);
+const View = forwardRef(
+	(
+		{ className, children }: PropsWithChildren<{ className: string }>,
+		ref,
+	) => {
```

Comment:
Question: partly out of scope, but is there a particular reason this component needs to use `forwardRef`? Don't see any consumers passing in refs.

Reply:
I don't think there is. I'll go ahead and convert it to a standard functional component.